test/integration/websocket_integration_test.cc

```diff
@@ -440,13 +440,15 @@ TEST_P(WebsocketIntegrationTest, WebsocketCustomFilterChain) {
     codec_client_->sendData(encoder_decoder.first, large_req_str, false);
     ASSERT_TRUE(response_->waitForEndStream());
     EXPECT_EQ("413", response_->headers().getStatusValue());
-    waitForClientDisconnectOrReset();
+    if (downstreamProtocol() != Http::CodecType::HTTP3) {
```

Comment:
I understand the second change in this file (since h2 and h3 mostly act the same). But I don't think I understand this. Why does the client behave different in h3?

Reply:
Discussed offline, will track down in https://github.com/envoyproxy/envoy/issues/26323